utils/checks/snapshots/features.yaml

```diff
@@ -247,6 +247,11 @@
     - default
     - sha2
     - std
+- name: bstr 0.2.17
```

Comment:
Where does this come from?

Reply:
If I remember correctly, it has been added after using `testing` feature at precompile code.
<img width="655" alt="image" src="https://github.com/humanode-network/humanode/assets/39522748/8b8322c5-71df-4d2a-9236-2b41f2522781">